go.mod

```diff
@@ -246,9 +246,12 @@ require (
 	github.com/coreos/go-systemd/v22 v22.5.0 // indirect
 	github.com/cpuguy83/go-md2man/v2 v2.0.6 // indirect
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
+	github.com/diskfs/go-diskfs v1.6.0 // indirect
```

Comment:
why is this indirect? missed a `go mod tidy`?

Reply:
Ah yes, you're exactly right 🤦